appveyor.yml

```diff
@@ -18,7 +18,19 @@ build_script:
 
     $id = $id.Substring(0, 13)
 
-    .\pack.cmd ci-$id
+    cmd /c call pack.cmd ci-$id
+
+    if ($LASTEXITCODE -ne 0) {
+        throw "Building/Packing failed with an exit code of $LASTEXITCODE."
+    }
+
+    $diff = git diff --exit-code 2>&1
```

Comment:
Adding `--ignore-all-space` should reduce the false positive rate significantly.

Reply:
Not sure how this helps the overall situation. The whitespace noise isn't a major issue at this point as PR merges on top of `master` failing. 😕